omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java

```diff
@@ -94,6 +89,33 @@ private void CreateApiDefinition() {
 		swaggerSpecification.setConsumes(consumes);
 	}
 	
+	private RequestContext BuildDummyRequestContextForOperationTest(OperationEnum operation) {
+		RequestContext rc = new RequestContext();
+		rc.setRequest(((ServletRequestAttributes) RequestContextHolder.getRequestAttributes()).getRequest());
+		
+		switch (operation) {
+			case get:
+				rc.setLimit(1);
+				
+		}
+		
+		return rc;
+	}
+	
+	private boolean TestOperationImplemented(OperationEnum operation, BaseDelegatingResource resource) {
+		switch (operation) {
+			case get:

```

Comment:
i would imagine you would do this for all operations because can chose not to support any operation